00XX:reassert-fee-payment-method

```diff
@@ -0,0 +1,54 @@
+## HIP: Driving Value Into The IOT Network Through IOT Token Payments
+- Author: @maxgold91 
+- Start Date: 2023-04-17
+- Category: Technical
+- Status: Draft
+- Original PR:
+- Tracking Issue:
+- Vote Requirements: 
+
+## Summary
+
+This HIP proposes changing the method of payment for Location Assert Fees from HNT tokens to IOT tokens. The primary purpose of this change is to drive value into IOT tokens and provide them with a small amount of utility. This HIP does not propose altering any other aspect of Location Assert Fees. The proposed changes are intended to be implemented in compliance with the rules set forth in [HIP-51](https://github.com/helium/HIP/blob/main/0051-helium-dao.md).
+
+## Motivation
+
+Currently, Location Assert Fees are paid using HNT tokens. However, there is no incentive to pay for onboarding using HNT under HIP-51, as none of the burned tokens accrue back to the subDAO. This proposal seeks to address this issue by transitioning the method of payment to IOT tokens.
```

Comment:
..." paid using HNT tokens which are burned for Data Credits (DC), and then subsequently used to pay the Location Assert Fees." ...

Reply:
I would also add here some more context for why you're proposing this:
- what are you seeing about the tokenomics that prompted you to write this
- what's not going right with the tokenomics
- what's the problem with IOT/assert fees/ the use of DC,etc

^ mostly prompting you here to say more.